charts/csi-isilon/values.yaml

```diff
@@ -35,7 +35,7 @@ images:
 # CSI driver log level
 # Allowed values: "error", "warn"/"warning", "info", "debug"
 # Default value: "debug"
```

Comment:
default value is `info`. Please update the comment.
Applies to all other places/files.